packages/data-service-generator/src/server/create-dotenv.ts

```diff
@@ -75,3 +77,21 @@ function removeDuplicateKeys(arr: VariableDictionary): VariableDictionary {
   });
   return Array.from(variablesMap, ([key, value]) => ({ [key]: value }));
 }
+
+function sortAlphabetically(arr: VariableDictionary): VariableDictionary {
```

Comment:
This function seems duplicated. It might make more sense to export this from a unique place (so also unit tests won't be duplicated)

Reply:
moved to `utils/dotenv.ts` is it ok?